packages/cli/src/commands/env/pull.ts

```diff
@@ -81,7 +81,7 @@ export default async function pull(
   }
 
   output.print(
-    `Downloading Development Environment Variables for Project ${chalk.bold(
+    `Downloading "${environment}" Environment Variables for Project ${chalk.bold(
```

Comment:
Do we need to use `title(environment)` here so the first letter becomes uppercase?

Reply:
We could, but I like that the environment is quoted to make it more obvious that it could have been something else.